### PR TITLE
Cdd 1912

### PR DESCRIPTION
**Pull request by googley42**

### CODE REVIEW

File: test/util/CustomsDeclarationsExternalServicesConfig.scala [hunk 1]
```diff
@@ -27,6 +27,7 @@ object CustomsDeclarationsExternalServicesConfig {
   val MdgStatusDeclarationServiceContextV3 = "/mdgStatusDecService3/submit3"
   val ApiSubscriptionFieldsContext = "/api-subscription-fields/field"
   val GoogleAnalyticsContext = "/send-google-analytics"
+  val CustomsDeclarationsMetricsContext = "/log-time"
```
Comment (mattclark-zerogravit) [inline test/util/CustomsDeclarationsExternalServicesConfig.scala]: This has changed to 'log-times'

**Comment by paulblakehmrc** [comment]
Is leaving out the metrics connector call deliberate? There is some work required to capture the entry timestamp and carry it through until the exit point where the connector is called.

File: test/util/CustomsDeclarationsMetricsTestData.scala [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import java.time.{ZoneId, ZonedDateTime}
+
+import uk.gov.hmrc.customs.declaration.model.CustomsDeclarationsMetricsRequest
+
+object CustomsDeclarationsMetricsTestData {
+
+  val UtcZoneId: ZoneId = ZoneId.of("UTC")
+  val EventStart: ZonedDateTime = ZonedDateTime.of(2015, 11, 30, 23, 45,
+    59, 0, UtcZoneId)
+  val EventEnd: ZonedDateTime = EventStart.plusSeconds(2)
```
Comment (mattclark-zerogravit) [inline test/util/CustomsDeclarationsMetricsTestData.scala]: This is probably ok to start with but we do need this placing so we get a valid endtime. Maybe the start & end  times need adding to the request context so they are available at the last step ready to be sent to the metrics service

Comment (googley42) [reply test/util/CustomsDeclarationsMetricsTestData.scala]: this PR is just for connector. Plenty more code to come around action builder pipeline etc

Comment (mattclark-zerogravit) [reply test/util/CustomsDeclarationsMetricsTestData.scala]: ok I'll accept then

File: test/util/CustomsDeclarationsExternalServicesConfig.scala (hunk 1 above)
Comment (googley42) [reply test/util/CustomsDeclarationsExternalServicesConfig.scala]: fixed

**Pull request merged**